compat/v0/compat.go

```diff
@@ -68,8 +68,8 @@ func ShardsResponseFromState(state jsonrpc.ResponseQueryState) (ResponseQuerySha
 	return resp, nil
 }
 
-// TxFromV1Tx takes a V1 Tx and converts it to a V0 Tx
-func TxFromV1Tx(t tx.Tx, hash B32, hasOut bool, bindings txengine.Bindings) (Tx, error) {
+// TxFromV1Tx takes a V1 Tx and converts it to a V0 Tx, the given tx has to be a mint tx.
```

Comment:
Hmm, let me double check this, might be problematic if we only support mint txes

Reply:
I believe this has been fixed in #119